internal/cri/server/container_create.go

```diff
@@ -362,6 +373,99 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	return &runtime.CreateContainerResponse{ContainerId: id}, nil
 }
 
+func (c *criService) mutateMounts(
+	ctx context.Context,
+	platform imagespec.Platform,
+	extraMounts []*runtime.Mount,
+) error {
+	for _, m := range extraMounts {
+		err := c.mutateOCIBindMount(ctx, platform, m)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (c *criService) mutateOCIBindMount(
+	ctx context.Context,
+	platform imagespec.Platform,
+	extraMount *runtime.Mount,
+) (retErr error) {
+	imageSpec := extraMount.GetImage()
+	if imageSpec == nil {
+		return nil
+	}
+
+	ref := imageSpec.GetImage()
```

Comment:
https://github.com/kubernetes/cri-api/blob/c20fa403eb24e7775cd6c41eb37c696ed4a26a5c/pkg/apis/runtime/v1/api.pb.go#L3557-L3570

This field is supposed to be imageDigest in the comments, but the cri-o implementation seems to use it as imageName, so it's the same here